profile/README.md

```diff
@@ -0,0 +1,26 @@
+![OpenSearch Banner](Banner.jpg)
+
+OpenSearch is a community-driven, Apache 2.0-licensed open source search and analytics suite that makes it easy to ingest, search, visualize, and analyze data. Developers build with OpenSearch for use cases such as application search, log analytics, data observability, data ingestion, and more.
+
+## Contributing ✍️
+
+We are built 🧱 by the community for the community. If you would like to contribute there are many ways: 
+
+- 👀 Check out a project's `CONTRIBUTING.md` file to learn how to contribute. 
+- [Blogs](https://github.com/opensearch-project/project-website) and [Documentation Updates](https://github.com/opensearch-project/documentation-website)!
+- Finally, make sure to try [OpenSearch](https://opensearch.org/docs/latest/opensearch/install/docker/) 🔎, [OpenSearch Dashboards](https://playground.opensearch.org/app/home) 🖥, and our Plugins/Client Libraries 📚 and leave feedback! 
+
+
+## Get Involved in our Community!
+
+There are several places where our community meets make sure to check them out!
+
+- 📝 [Forum](https://forum.opensearch.org/)
+- ▶️ [YouTube](https://www.youtube.com/c/OpenSearchProject)
+- 🐤 [Twitter](https://twitter.com/OpenSearchProj)
+- 🧑‍💼 [LinkedIn](https://www.linkedin.com/company/opensearch-project/)
+
```

Comment:
Remove double-space?

Reply:
Resolved, its a personal preference but not necessary.